opentargets_pharmgkb/pandas_utils.py

```diff
@@ -10,7 +10,7 @@ def none_to_nan(x):
     return np.nan if x is None else x
 
 
-def explode_column(df, source_col, target_col, sep=';'):
+def explode_column(df, source_col, target_col, sep=';', split_only=False):
```

Comment:
I'm being a bit picky here but running `explode_column` without exploding seems odd.

Reply:
Or is it that `explode_column` means split column and it can be split into a list or into many rows.

Reply:
Yeah this kind of bothered me too... what about naming it `split_and_explode_column`? This way `split` == split the string into a list and `explode` == explode the list value into rows

Reply:
Sounds reasonable